migrations/20220324080020-add-place-comment.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      await queryInterface.createTable(
```

Comment:
I am pretty sure we also want to have updatedAt date in the database. Could you add it?

Reply:
Migration ads both fields.